### PR TITLE
Add Bulgarian translation.

### PR DESCRIPTION
**Pull request by GriffinMLN**
Hey :)
I followed the template were outlined - I did the translation and added the country flag to the table. I hope you approve!

If you decide to accept this PR, would you please add the '**hacktoberfest-accepted**' **label**? Thank you in advance!

### CODE REVIEW

**Comment by sabesansathananthan** [comment]
@esau-morais Please review it

File: translations/README.bg.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# GitHub Резюме Генератор
+
+![React GitHub Resume](./src/assets/readme/screenshot.png)
```
Comment (esau-morais) [inline translations/README.bg.md:3]: Please, change this route to `../src/assets/readme/screenshot.png`

File: translations/README.bg.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Съдържание
+
+- [Относно](#about)
```
Comment (esau-morais) [inline translations/README.bg.md:7]: Once time you change the topic name, you need to change the links too

File: translations/README.bg.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+4. Посетете <http://localhost:3000>
+
+## [Демо на живо](https://react-github-resume.vercel.app/)
+
+[![Deploy с Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 Лиценз
+
+Този проект е лицензиран под лиценза MIT - вижте файла [LICENSE](./LICENSE) за подробности.
```
Comment (esau-morais) [inline translations/README.bg.md:86]: Please, just change the `LICENSE` word to the right language

File: translations/Translations.md [hunk 4]
```diff
@@ -11,3 +11,4 @@
...
 <img alt="Italiano" title="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">                                                                                                                                                 | [Italiano](./README.it.md)                                               |
 | <img alt="Čeština" title="Čeština" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cz.svg" width="22">                                                                                                                                               | [Čeština](README.cs.md)                                              |
+| <img alt="Български" title="Български" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/bg.svg" width="22">                                                                                                                                               | [Български](README.bg.md)                                              |
```
Comment (esau-morais) [inline translations/Translations.md:14]: Please, add the `./README.bg.md` instead just `README.bg.md`

**Comment by esau-morais** [comment]
@GriffinMLN , after make the required changes, don't forget to solve the conflicts.

_Note: Notice two things:_

- [ ] Be careful to not delete an recently added language (every time is coming more)
- [ ] It's not necessary add the label `hacktoberfest-accepted` to your PR, once time the repository already is with the required topic (`hacktoberfest`)

**Comment by sabesansathananthan** [comment]
@GriffinMLN  Resolve the conflicts

File: translations/Translations.md (hunk 4 above)
Comment (GriffinMLN) [reply translations/Translations.md]: Fixing.

Btw, I copied the one above, so as you can see this means that `README.cs.md` is wrong as well.

File: translations/README.bg.md (hunk 2 above)
Comment (GriffinMLN) [reply translations/README.bg.md]: I looked at the Russian Readme for reference btw and none of this is done there as well :/
I'll adjust it now.

File: translations/README.bg.md (hunk 3 above)
Comment (GriffinMLN) [reply translations/README.bg.md]: Fixing.
Again, you might want to fix the Russian version.

File: translations/README.bg.md (hunk 1 above)
Comment (GriffinMLN) [reply translations/README.bg.md]: Fixing. FIY the original English readme uses `./src/assets/readme/screenshot.png`

Comment (esau-morais) [reply translations/README.bg.md]: Sorry, it was a mistake. But the right is how I reviewed

File: translations/README.bg.md (hunk 2 above)
Comment (esau-morais) [reply translations/README.bg.md]: Thank you for noticing, I'll see that

File: translations/README.bg.md (hunk 3 above)
Comment (esau-morais) [reply translations/README.bg.md]: Thank you again for noticing, I'll review that

File: translations/Translations.md (hunk 4 above)
Comment (esau-morais) [reply translations/Translations.md]: Yeah, I noticed it. I'll review it

**Comment by GriffinMLN** [comment]
@esau-morais do you need me to adjust anything else?

**Comment by esau-morais** [comment]
> @esau-morais do you need me to adjust anything else?

No, I think it's all good now. Thank you

**Comment by sabesansathananthan** [comment]
Please resolve conflicts. Sorry for the inconvenience. We fixed some bugs in the previous PRs. Could you please get the latest pull from the upstream and resolve conflicts.

**Comment by GriffinMLN** [comment]
Sorry, I'm not as advanced and I'm having troubles with fetch/pull to actually get in line with the master branch, so I'll just have to start from scratch and open a new pull request. I hope this is okay with you. Sorry for the inconvenience.

**Pull request closed without merging**